test/parallel/test-util-format.js

```diff
@@ -553,3 +553,37 @@ assert.strictEqual(
     message: /"inspectOptions".+object/
   });
 });
+
+assert.strictEqual(
+  util.formatWithOptions({ colors: true }, '%cfoo', 'color: red'),
+  '\x1b[31mfoo\x1b[0m'
```

Comment:
It would be more readable to use `util.styleText` here

Reply:
After trying that 👆 it looks like util.styleText() uses ...93m sometimes to just reset the foreground color instead of always using ...0m to reset everything (which %c does) which causes the tests to fail (strings not equal).